mautic/mautic
# Frequency Rule transactional email bypass

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | n
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds support to transactional emails for campaigns introduced in version 2.1.  Basically if it's a transactional email the frequency rules should be bypassed.

#### Steps to test this PR:
1. Apply this PR
2. create a campaign with a transactional email, email should be sent to contact regardless of their frequency rules.
3. You can also check that Marketing campaigns should still respect the frequency rules on the contacts.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. create a campaign with a send email action with a transactional email selected. If there is a frequency rule on the contact or set globally in the configuration settings, the transactional email checks to see if the rules are met, if they are then the email is not sent. When the purpose of the transactional email being, it needs to be sent regardless.
